Trakt/Api/Enums/TraktAction.cs

```diff
@@ -0,0 +1,28 @@
+#pragma warning disable SA1300
+
+using System.Text.Json.Serialization;
+
+namespace Trakt.Api.Enums
+{
+    /// <summary>
+    /// Enum TraktHdr.
+    /// </summary>
+    [JsonConverter(typeof(JsonStringEnumConverter))]
```

Comment:
You shouldn't need this, that converter is already added to the default `JsonOptions`